AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -436,6 +436,7 @@ thehouseofportable.com##body > #modalDisclaimer + [style^="display: block;"]:not
 thehouseofportable.com#%#AG_onLoad(function() { window.openInNewTab = function() {}; });
 thehouseofportable.com#%#(function(){var c=window.setTimeout;window.setTimeout=function(f,g){return !Number.isNaN(g)&&/urls\[|stt/.test(f.toString())&&(g*=0.01),c.apply(this,arguments)}.bind(window)})();
 genshin.gamedot.org#%#//scriptlet("prevent-setTimeout", "targetObj.html().length<")
+playonlinux.com#%#//scriptlet("abort-current-inline-script", "$", "load")
```

Comment:
You forgot about content blockers.
`playonlinux.com##.login > div[style*="background-color:#666;"][style*="color:#FFF"]` is ok.